src/vtquery.cpp

```diff
@@ -541,44 +476,50 @@ struct Worker : Nan::AsyncWorker {
 
         // number of results to loop through
         v8::Local<v8::Object> results_object = Nan::New<v8::Object>();
-        v8::Local<v8::Array> features_array = Nan::New<v8::Array>(results_queue_.size());
+        v8::Local<v8::Array> features_array = Nan::New<v8::Array>();
```

Comment:
unable to pre-allocate the size of this vector now that we have a default constructor and we've reservation for results_queue_